src/source/zero.rs

```diff
@@ -1,7 +1,5 @@
-use crate::Sample;
-use crate::Source;
-use std::marker::PhantomData;
-use std::time::Duration;
+use crate::{Sample, Source};
+use std::{marker::PhantomData, time::Duration};
```

Comment:
I don't like this change, can you keep it? The sample, source change is fine, just issues with the second line.